manifests/user-facing-clusterroles.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+  name: zalando-postgres-operator:users:edit
+rules:
+- apiGroups:
+  - acid.zalan.do
+  resources:
+  - postgresqls
+  - postgresqls/status
```

Comment:
`OperatorConfiguration` CRD is missing under `resources`. Any reason?

Reply:
Hmm. This is user-facing. I don't think users of the k8s cluster that want to create a database need to see the operator config in order to create a database, I consider that private to the operator and appropriate to stay at clusteradmin level permission to see.

Reply:
I thought about that, too. Fine for me.
I think, `postgresqls/status` could also be removed from the edit role. For admin and view it makes sense.

Reply:
Agreed. Updated. :)